Tests/Marketplace/marketplace_services.py

```diff
@@ -2435,6 +2467,54 @@ def get_packs_statistics_dataframe(bq_client):
     return packs_statistic_table
 
 
+def filter_packs_from_before_3_months(pack_list_to_filter: list, index_folder_path: str) -> List[str]:
+    """
+    Filtering packs from 'pack_list_to_filter' that were created more than 3 months ago by checking in the index file
+    Args:
+        pack_list_to_filter: The list of packs sorted by download rate to filter by creation date.
+        index_folder_path: The path in which the index.zip file was unzipped into.
+
+    Returns:
+        A list with pack names that were created within the last 3 months.
+    """
+    index_packs = {os.path.basename(pack_path): Pack(os.path.basename(pack_path), pack_path) for pack_path in
+                   glob.glob(f'{index_folder_path}/*')}
```

Comment:
If I'm not mistaken, `index.json` is under this directory's path, what happens in that case?

Reply:
Nothing, it will create a Pack instance by the name of `index.json`, which will never be selected as I'm selecting packs from `index_packs`, only if a pack by that name exists in `pack_list_to_filter` which contains only valid pack names.